### DISCUSSION TITLE
[BUG] Failed to open (invalid plots)

### DISCUSSION THREAD

**Discussion by maximus1218** [Chia-Network/chia-blockchain#2944] [Q&A]
After every plot gets finished, I am unable to farm the plot file. I have checked my keys, changed my plotting tactics, yet every single plot is invalid and unable to farm. What causes this to happen and how should I go about fixing it? 

I am on windows OS, plotting on an ssd, and putting plot files on an external USB. 

I also noticed that the plot files are completing the 4th phase in the log, and are remaining loaded at 100% for about five hours before they all end up in the Invalid Plots list.

**Comment by GASCAGO** [comment]
Same here. I just installed latest version of Chia as of today but no help.... In my case the plots never sync.... But before that I needed to rename them manually as they were copied to the farm drive as .2.tmp


![image](https://user-images.githubusercontent.com/83317152/116322825-14b22500-a77a-11eb-9e36-8ebb251aed26.png)
